### DISCUSSION TITLE
Formality of translations

### DISCUSSION THREAD

**Discussion by Tmpod** [mastodon/mastodon#18200] [General]
Hello!

I'd like to fully translate the new official mobile apps into Portuguese, but I have to clarify something first - what is the preferred style for translations? Formal or informal?
I tried searching about this but couldn't find any i18n guidelines, and discussions weren't active on the mastodon-android and mastodon-ios repos, so I ended up here, hopefully it's alright :3 

Thanks in advance!

**Comment by Gargron** [comment]
Probably informal!
